---
date: "Last updated `r format(Sys.Date())`"
output: html_document
params:
  pheno: "50"
  datfile: "../results/ukbb_h2part.RData"
title: "Heritability for `r params$pheno`"
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
require(plotly)
require(reshape2)

# handle params
phen <- as.character(params$pheno)
load(file=params$datfile)

# setup
dat_sub <- dat[which(dat$phenotype==phen),]

ukcode <- strsplit(phen,"_")[[1]][1]

isICD <- is.na(as.numeric(ukcode))
if(isICD){
	ukb_url = "http://biobank.ctsu.ox.ac.uk/crystal/search.cgi?srch=41202"
	ukcode = "41202"
}else{
	ukb_url = paste0("http://biobank.ctsu.ox.ac.uk/crystal/search.cgi?srch=",ukcode)
}

isBinary <- !is.na(dat_sub$N_case[1])

if(isBinary){
	Neff <- 4/((1/dat_sub$N_case[1])+(1/dat_sub$N_control[1]))
}else{
	Neff <- dat_sub$N[1]
}
isSmallCont <- (Neff < 1000 && !isBinary)
isSmallBinary <- (Neff < 1000 && isBinary)

format_bigint <- function(n){
	formatC(n,format="f",big.mark=',',drop0trailing=T)
}

```
<br>

<div class="panel panel-primary">
<div class="panel-body">
### **Phenotype:** `r dat_sub$description`

#### **Estimated SNP heritability:** `r sprintf("%0.4f",dat_sub$h2_liability)` (se=`r sprintf("%0.3g",dat_sub$h2_liability_se)`, p=`r sprintf("%0.3e",dat_sub$h2_p)`)

This phenotype can be found `r if(isICD){" as part of the ICD code listings"}` on the UK Biobank Showcase for [code `r ukcode`](`r ukb_url`). GWAS results are available for _N_ = `r format_bigint(dat_sub$N)` unrelated individuals of European ancestry. This is a `r ifelse(isBinary, "binary", "continuous")` phenotype`r if(isBinary){paste0(" with ",format_bigint(dat_sub$N_case)," cases and ",format_bigint(dat_sub$N_control)," controls")}`.  

</div>
</div>

`r if(isBinary){paste0("<div class=\"panel panel-info\"><div class=\"panel-body\"> <p class=\"text-info\">**Note:**</p> Heritability for this binary outcome is reported on the liability scale, assuming that the population prevalence matches the observed prevalence of ",sprintf("%0.3g",dat_sub$prevelence)," in the UK Biobank analysis set. This may be unreliable if the outcome was only defined for a subset of individuals, or if the the UK Biobank study population (which skews older, whiter, and higher SES than the general UK population, <a href=\"https://academic.oup.com/aje/article/doi/10.1093/aje/kwx246/3883629/Comparison-of-Sociodemographic-and-Health-Related\">among other things</a>) is not a representative sample for this phenotype. For comparison, estimated **observed-scale heritability** is: ", sprintf("%0.4f",dat_sub$h2_observed)," (se=",sprintf("%0.3g",dat_sub$h2_observed_se),"). The p-value is unaffected.</div></div>")}`
`r if(isSmallCont){paste0("<div class=\"panel panel-warning\">Warning<div class=\"panel-body\"> The sample size for this phenotype is quite limited. Estimates are likely to be unstable. </div></div>")}`
`r if(isSmallBinary){paste0("<div class=\"panel panel-warning\"><div class=\"panel panel-heading\"><h3 class=\"panel-title\">Warning</h3></div><div class=\"panel-body\"> The effective sample size for this phenotype is quite limited (N_eff=",format_bigint(round(Neff,2)),"). Estimates are likely to be unstable. Liability-scale transformation for this binary phenotype may be especially misleading. </div></div>")}`

<br>

### Full Results

All results are from partitioned heritability analysis of this phenotype using LD score regression ([Bulik-Sullivan et al. 2015](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4495769/), [github repo](https://github.com/bulik/ldsc)) with 74 annotations as described by [Gazal et al. 2017](http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3954.html) (also on [biorxiv](http://www.biorxiv.org/content/early/2017/04/03/082024)). See [Technical Details](`details.html`) for more information.

<br>

#### Confounding and Model Misspecification

In addition to heritability, LD score regression also estimates a intercept term that indexes of population stratification, other counfounding, and potential misspecification in the partitioned LD score model for the distribution of genetic effects genome-wide. 

* **Intercept:** `r round(dat_sub$intercept,4)` (se=`r sprintf("%0.3g",dat_sub$intercept_se)`, p=`r sprintf("%0.3e",dat_sub$intercept_p)`)
* **Mean $\chi^2$:** `r round(dat_sub$mean_chi2,4)`
* **$\lambda_{GC}$:** `r round(dat_sub$lambdaGC,4)`
* **Ratio:** `r sprintf("%0.4f",dat_sub$ratio)` (se=`r sprintf("%0.4f",dat_sub$ratio_se)`)

Intercept values near 1 indicate little or no confounding. Comparing the intercept and the mean $\chi^2$ statistic provides an rough index for how much of the polygenic signal in the GWAS may be due to confounding rather than genetic effects (assuming the LD score model is well specified).

<br>

#### Partitioned Heritability

##### Significance of categorical annotations

```{r part_coef_p, echo=FALSE}
coef_p_cols <- names(dat_sub)[grep("..Coefficient_p",names(dat_sub))]

# remove continuous annotations
coef_p_cols <- coef_p_cols[-grep("base",coef_p_cols)]
coef_p_cols <- coef_p_cols[-grep(".extend.500",coef_p_cols)]
coef_p_cols <- coef_p_cols[-grep("MAF",coef_p_cols)]
coef_p_cols <- coef_p_cols[-grep("Recomb",coef_p_cols)]
coef_p_cols <- coef_p_cols[-grep("GERP.NS",coef_p_cols)]
coef_p_cols <- coef_p_cols[-grep("Backgrd",coef_p_cols)]
coef_p_cols <- coef_p_cols[-grep("CpG",coef_p_cols)]
coef_p_cols <- coef_p_cols[-grep("Diversity",coef_p_cols)]

dtmp <- dat_sub[,c("phenotype",coef_p_cols)]
df <- reshape2::melt(dtmp,id.vars = "phenotype")
df$logp <- -log10(df$value)
df$variable <- sapply(df$variable,function(a) strsplit(as.character(a), split="\\.\\.")[[1]][1])

# define barplot
pp <- plot_ly(df, width=800) %>%
			add_trace(
			  x=~variable,
			  y=~logp,
			  hoverinfo="text",
			  text=~paste0("Annotation: ",variable,
			  		   "<br>p-value: ", signif(value, 3)),
			  type="bar",
			  name="p-value for coefficient"
	  ) %>% add_trace(
	  		x=~variable, 
	  		y=rep(-log10(.05/38),nrow(df)), 
	  		type="scatter",
	  		mode="lines",
	  		hoverinfo="text",
			text="",
	  		name="Bonferroni significance",
	  		yaxis='y2'
	  ) %>% add_trace(
	  		x=~variable, 
	  		y=rep(-log10(.05),nrow(df)), 
	  		type="scatter",
	  		mode="lines",
	  		hoverinfo="text",
			text="",
	  		name="Nominal significance",
	  		yaxis='y2'
	  ) %>% layout(
	  			 showlegend=F,
	  			 xaxis = list(title="Annotation", tickangle=-30),
	  			 yaxis = list(title="-log10(p-value)",side='left', range=c(0,max(df$logp))),
	  			 yaxis2 = list(title="", showgrid = FALSE, zeroline = FALSE, overlaying='y', side='right', range=c(0,max(df$logp)),showticklabels=F),
	  			 margin=list(b=150,l=100)
	  ) 

config(pp, collaborate = F, showLink=F, displayModeBar=F, displaylogo=F, sendData=F)

```

The orange line indicates Bonferroni-corrected significance across 38 annotations for this phenotype ($p <$ 1.3e-3). The green line indicates nominal significance ($p <$ .05). Bonferroni correction excludes the 10 MAF bin annotations and the 26 buffer annotations (extending 500-bp around the baseline functional annotations) since they are primarily intended as statistical controls rather than for direct inference.
