<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Heritability for K59</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<script src="site_libs/htmlwidgets-0.9/htmlwidgets.js"></script>
<script src="site_libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/crosstalk-1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.1/js/crosstalk.min.js"></script>
<link href="site_libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<link rel="apple-touch-icon-precomposed" sizes="57x57" href="apple-touch-icon-57x57.png" />
<link rel="apple-touch-icon-precomposed" sizes="114x114" href="apple-touch-icon-114x114.png" />
<link rel="apple-touch-icon-precomposed" sizes="72x72" href="apple-touch-icon-72x72.png" />
<link rel="apple-touch-icon-precomposed" sizes="144x144" href="apple-touch-icon-144x144.png" />
<link rel="apple-touch-icon-precomposed" sizes="60x60" href="apple-touch-icon-60x60.png" />
<link rel="apple-touch-icon-precomposed" sizes="120x120" href="apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon-precomposed" sizes="76x76" href="apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon-precomposed" sizes="152x152" href="apple-touch-icon-152x152.png" />
<link rel="icon" type="image/png" href="favicon-196x196.png" sizes="196x196" />
<link rel="icon" type="image/png" href="favicon-96x96.png" sizes="96x96" />
<link rel="icon" type="image/png" href="favicon-32x32.png" sizes="32x32" />
<link rel="icon" type="image/png" href="favicon-16x16.png" sizes="16x16" />
<link rel="icon" type="image/png" href="favicon-128.png" sizes="128x128" />
<meta name="application-name" content="&nbsp;"/>
<meta name="msapplication-TileColor" content="#FFFFFF" />
<meta name="msapplication-TileImage" content="mstile-144x144.png" />
<meta name="msapplication-square70x70logo" content="mstile-70x70.png" />
<meta name="msapplication-square150x150logo" content="mstile-150x150.png" />
<meta name="msapplication-wide310x150logo" content="mstile-310x150.png" />
<meta name="msapplication-square310x310logo" content="mstile-310x310.png" />


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-106608402-1', 'auto');
  ga('send', 'pageview');

</script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 61px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 66px;
  margin-top: -66px;
}

.section h2 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h3 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h4 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h5 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h6 {
  padding-top: 66px;
  margin-top: -66px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">UKBB Heritability</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="h2_browser.html">Browser</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Plots
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="viz_h2.html">Heritability and Intercept</a>
    </li>
    <li>
      <a href="viz_annot.html">Partitioned Annotations</a>
    </li>
    <li>
      <a href="viz_qq.html">QQ Plots</a>
    </li>
    <li>
      <a href="viz_sampsize.html">Sample Size Scaling</a>
    </li>
    <li>
      <a href="viz_univar.html">Univariate LDSR</a>
    </li>
  </ul>
</li>
<li>
  <a href="details.html">Technical Details</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/Nealelab/UKBB_ldsc">Github Repo</a>
</li>
<li>
  <a href="http://www.nealelab.is/blog/2017/7/19/rapid-gwas-of-thousands-of-phenotypes-for-480000-samples-in-the-uk-biobank">Neale Lab UKBB</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Heritability for K59</h1>
<h4 class="date"><em>Last updated 2017-09-18</em></h4>

</div>


<p><br></p>
<div class="panel panel-primary">
<div class="panel-body">
<h3 id="phenotype-diagnoses---main-icd10-k59-other-functional-intestinal-disorders"><strong>Phenotype:</strong> Diagnoses - main ICD10: K59 Other functional intestinal disorders</h3>
<h4 id="estimated-snp-heritability-0.0259-se0.0383-p2.495e-01"><strong>Estimated SNP heritability:</strong> 0.0259 (se=0.0383, p=2.495e-01)</h4>
<p>This phenotype can be found as part of the ICD code listings on the UK Biobank Showcase for <a href="http://biobank.ctsu.ox.ac.uk/crystal/field.cgi?id=41202">code 41202</a>. GWAS results are available for <em>N</em> = 337,199 unrelated individuals of European ancestry. This is a binary phenotype with 2,419 cases and 334,780 controls. <br></p>
<p><em>Note:</em> Heritability for this binary outcome is reported on the liability scale, assuming that the population prevalence matches the prevalence in the UK Biobank analysis set (0.00717). This may be unreliable if the outcome was limited to a subset of individuals, or if the UK Biobank study population is not representative for this phenotype (<a href="https://academic.oup.com/aje/article/doi/10.1093/aje/kwx246/3883629/Comparison-of-Sociodemographic-and-Health-Related">which is likely in many cases</a>).<br><br>Estimated <strong>observed-scale heritability</strong>: 0.0014 (se=0.00213).</p>
</div>
</div>
<p><br></p>
<div class="well">
<h3 id="additional-results">Additional Results</h3>
<p>All results are from partitioned heritability analysis of this phenotype using LD score regression (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4495769/">Bulik-Sullivan et al. 2015</a>, <a href="https://github.com/bulik/ldsc">github repo</a>) with 74 annotations as described by <a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3954.html">Gazal et al. 2017</a> (also on <a href="http://www.biorxiv.org/content/early/2017/04/03/082024">biorxiv</a>). See <a href="%60details.html%60">Technical Details</a> for more information.</p>
<p><br></p>
<h4 id="confounding-and-model-misspecification">Confounding and Model Misspecification</h4>
<p>In addition to heritability, LD score regression also estimates a intercept term that indexes of population stratification, other counfounding, and potential misspecification in the partitioned LD score model for the distribution of genetic effects genome-wide.</p>
<ul>
<li><strong>Intercept:</strong> 0.9992 (se=0.00906, p=5.356e-01)</li>
<li><strong>Mean <span class="math inline">\(\chi^2\)</span>:</strong> 1.0084</li>
<li><strong><span class="math inline">\(\lambda_{GC}\)</span>:</strong> 1.0097</li>
<li><strong>Ratio:</strong> -0.0967 (se=1.0823)</li>
</ul>
<p>Intercept values near 1 indicate little or no confounding. Comparing the intercept and the mean <span class="math inline">\(\chi^2\)</span> statistic provides an rough index for how much of the polygenic signal in the GWAS may be due to confounding rather than genetic effects (assuming the LD score model is well specified).</p>
<p><br></p>
<h4 id="partitioned-heritability">Partitioned Heritability</h4>
<h5 id="significance-of-categorical-annotations">Significance of categorical annotations</h5>
<div class="container-fluid crosstalk-bscols">
<div class="fluid-row">
<div class="col-xs-12">
<div id="dcff7cc93c" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="dcff7cc93c">{"x":{"visdat":{"dcff3e2b1cff":["function () ","plotlyVisDat"]},"cur_data":"dcff3e2b1cff","attrs":{"dcff3e2b1cff":{"alpha":1,"sizes":[10,100],"x":{},"y":{},"hoverinfo":"text","text":{},"type":"bar","name":"p-value for coefficient"},"dcff3e2b1cff.1":{"alpha":1,"sizes":[10,100],"x":{},"y":[2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079],"type":"scatter","mode":"lines","hoverinfo":"text","text":"","name":"Bonferroni significance","yaxis":"y2"},"dcff3e2b1cff.2":{"alpha":1,"sizes":[10,100],"x":{},"y":[1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398],"type":"scatter","mode":"lines","hoverinfo":"text","text":"","name":"Nominal significance","yaxis":"y2"}},"layout":{"margin":{"b":150,"l":100,"t":25,"r":10},"showlegend":false,"xaxis":{"domain":[0,1],"title":"Annotation","tickangle":-30,"type":"category","categoryorder":"array","categoryarray":["Coding_UCSC","Conserved_LindbladToh","CTCF_Hoffman","DGF_ENCODE","DHS_peaks_Trynka","DHS_Trynka","Enhancer_Andersson","Enhancer_Hoffman","FetalDHS_Trynka","GERP.RSsup4","H3K27ac_Hnisz","H3K27ac_PGC2","H3K4me1_peaks_Trynka","H3K4me1_Trynka","H3K4me3_peaks_Trynka","H3K4me3_Trynka","H3K9ac_peaks_Trynka","H3K9ac_Trynka","Intron_UCSC","Promoter_UCSC","PromoterFlanking_Hoffman","Repressed_Hoffman","Super_Enhancer_Vahedi","SuperEnhancer_Hnisz","TFBS_ENCODE","Transcr_Hoffman","TSS_Hoffman","Typical_Enhancer_Vahedi","UTR_3_UCSC","UTR_5_UCSC","WeakEnhancer_Hoffman"]},"yaxis":{"domain":[0,1],"title":"-log10(p-value)","side":"left","range":[0,1.69241169932026]},"yaxis2":{"title":"","showgrid":false,"zeroline":false,"overlaying":"y","side":"right","range":[0,1.69241169932026],"showticklabels":false},"hovermode":"closest"},"source":"A","config":{"modeBarButtonsToAdd":[],"cloud":false,"showLink":false,"displayModeBar":false,"displaylogo":false,"sendData":false},"data":[{"x":["Coding_UCSC","Conserved_LindbladToh","CTCF_Hoffman","DGF_ENCODE","DHS_peaks_Trynka","DHS_Trynka","Enhancer_Andersson","Enhancer_Hoffman","FetalDHS_Trynka","H3K27ac_Hnisz","H3K27ac_PGC2","H3K4me1_peaks_Trynka","H3K4me1_Trynka","H3K4me3_peaks_Trynka","H3K4me3_Trynka","H3K9ac_peaks_Trynka","H3K9ac_Trynka","Intron_UCSC","PromoterFlanking_Hoffman","Promoter_UCSC","Repressed_Hoffman","SuperEnhancer_Hnisz","TFBS_ENCODE","Transcr_Hoffman","TSS_Hoffman","UTR_3_UCSC","UTR_5_UCSC","WeakEnhancer_Hoffman","Super_Enhancer_Vahedi","Typical_Enhancer_Vahedi","GERP.RSsup4"],"y":[0.67831094515773,0.0347811893323698,0.063072105563976,0.190940468326441,0.0323131700955939,0.461627169474347,0.0563265977120921,0.375687771381738,0.837730203083673,1.12818788172766,0.915150253938265,0.114135008255878,0.017660470655303,1.66734463662032,0.198360240549701,0.199913154010352,0.0471758506033978,1.04470758828226,0.05387845009227,1.38301300077417,0.714642245495075,0.132719479335489,0.14523917872459,0.333624900127665,0.0529901484752844,0.0312177971872924,1.69241169932026,1.0960595496498,0.572353616341896,0.165128074990865,0.0552603897191569],"hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"text":["Annotation: Coding_UCSC<br>p-value: 0.21","Annotation: Conserved_LindbladToh<br>p-value: 0.923","Annotation: CTCF_Hoffman<br>p-value: 0.865","Annotation: DGF_ENCODE<br>p-value: 0.644","Annotation: DHS_peaks_Trynka<br>p-value: 0.928","Annotation: DHS_Trynka<br>p-value: 0.345","Annotation: Enhancer_Andersson<br>p-value: 0.878","Annotation: Enhancer_Hoffman<br>p-value: 0.421","Annotation: FetalDHS_Trynka<br>p-value: 0.145","Annotation: H3K27ac_Hnisz<br>p-value: 0.0744","Annotation: H3K27ac_PGC2<br>p-value: 0.122","Annotation: H3K4me1_peaks_Trynka<br>p-value: 0.769","Annotation: H3K4me1_Trynka<br>p-value: 0.96","Annotation: H3K4me3_peaks_Trynka<br>p-value: 0.0215","Annotation: H3K4me3_Trynka<br>p-value: 0.633","Annotation: H3K9ac_peaks_Trynka<br>p-value: 0.631","Annotation: H3K9ac_Trynka<br>p-value: 0.897","Annotation: Intron_UCSC<br>p-value: 0.0902","Annotation: PromoterFlanking_Hoffman<br>p-value: 0.883","Annotation: Promoter_UCSC<br>p-value: 0.0414","Annotation: Repressed_Hoffman<br>p-value: 0.193","Annotation: SuperEnhancer_Hnisz<br>p-value: 0.737","Annotation: TFBS_ENCODE<br>p-value: 0.716","Annotation: Transcr_Hoffman<br>p-value: 0.464","Annotation: TSS_Hoffman<br>p-value: 0.885","Annotation: UTR_3_UCSC<br>p-value: 0.931","Annotation: UTR_5_UCSC<br>p-value: 0.0203","Annotation: WeakEnhancer_Hoffman<br>p-value: 0.0802","Annotation: Super_Enhancer_Vahedi<br>p-value: 0.268","Annotation: Typical_Enhancer_Vahedi<br>p-value: 0.684","Annotation: GERP.RSsup4<br>p-value: 0.881"],"type":"bar","name":"p-value for coefficient","marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","line":{"color":"transparent"}},"xaxis":"x","yaxis":"y","frame":null},{"x":["Coding_UCSC","Conserved_LindbladToh","CTCF_Hoffman","DGF_ENCODE","DHS_peaks_Trynka","DHS_Trynka","Enhancer_Andersson","Enhancer_Hoffman","FetalDHS_Trynka","H3K27ac_Hnisz","H3K27ac_PGC2","H3K4me1_peaks_Trynka","H3K4me1_Trynka","H3K4me3_peaks_Trynka","H3K4me3_Trynka","H3K9ac_peaks_Trynka","H3K9ac_Trynka","Intron_UCSC","PromoterFlanking_Hoffman","Promoter_UCSC","Repressed_Hoffman","SuperEnhancer_Hnisz","TFBS_ENCODE","Transcr_Hoffman","TSS_Hoffman","UTR_3_UCSC","UTR_5_UCSC","WeakEnhancer_Hoffman","Super_Enhancer_Vahedi","Typical_Enhancer_Vahedi","GERP.RSsup4"],"y":[2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079],"type":"scatter","mode":"lines","hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"text":["","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],"name":"Bonferroni significance","yaxis":"y2","line":{"fillcolor":"rgba(255,127,14,1)","color":"rgba(255,127,14,1)"},"xaxis":"x","frame":null},{"x":["Coding_UCSC","Conserved_LindbladToh","CTCF_Hoffman","DGF_ENCODE","DHS_peaks_Trynka","DHS_Trynka","Enhancer_Andersson","Enhancer_Hoffman","FetalDHS_Trynka","H3K27ac_Hnisz","H3K27ac_PGC2","H3K4me1_peaks_Trynka","H3K4me1_Trynka","H3K4me3_peaks_Trynka","H3K4me3_Trynka","H3K9ac_peaks_Trynka","H3K9ac_Trynka","Intron_UCSC","PromoterFlanking_Hoffman","Promoter_UCSC","Repressed_Hoffman","SuperEnhancer_Hnisz","TFBS_ENCODE","Transcr_Hoffman","TSS_Hoffman","UTR_3_UCSC","UTR_5_UCSC","WeakEnhancer_Hoffman","Super_Enhancer_Vahedi","Typical_Enhancer_Vahedi","GERP.RSsup4"],"y":[1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398],"type":"scatter","mode":"lines","hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"text":["","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],"name":"Nominal significance","yaxis":"y2","line":{"fillcolor":"rgba(44,160,44,1)","color":"rgba(44,160,44,1)"},"xaxis":"x","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":[],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
</div>
</div>
</div>
<p>The orange line indicates Bonferroni-corrected significance across 38 annotations for this phenotype (<span class="math inline">\(p &lt;\)</span> 1.3e-3). The green line indicates nominal significance (<span class="math inline">\(p &lt;\)</span> .05). Bonferroni correction excludes the 10 MAF bin annotations and the 26 buffer annotations (extending 500-bp around the baseline functional annotations) since they are primarily intended as statistical controls rather than for direct inference.</p>
<p><br></p>
<h5 id="significance-of-continuous-annotations">Significance of continuous annotations</h5>
<div class="container-fluid crosstalk-bscols">
<div class="fluid-row">
<div class="col-xs-12">
<div id="dcff6c5da193" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="dcff6c5da193">{"x":{"visdat":{"dcff67f68fd7":["function () ","plotlyVisDat"]},"cur_data":"dcff67f68fd7","attrs":{"dcff67f68fd7":{"alpha":1,"sizes":[10,100],"x":{},"y":{},"hoverinfo":"text","text":{},"type":"bar","name":"p-value for coefficient"},"dcff67f68fd7.1":{"alpha":1,"sizes":[10,100],"x":{},"y":[2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079],"type":"scatter","mode":"lines","hoverinfo":"text","text":"","name":"Bonferroni significance","yaxis":"y2"},"dcff67f68fd7.2":{"alpha":1,"sizes":[10,100],"x":{},"y":[1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398],"type":"scatter","mode":"lines","hoverinfo":"text","text":"","name":"Nominal significance","yaxis":"y2"}},"layout":{"margin":{"b":150,"l":100,"t":25,"r":10},"showlegend":false,"xaxis":{"domain":[0,1],"title":"Annotation","tickangle":-30,"type":"category","categoryorder":"array","categoryarray":["Backgrd_Selection_Stat","CpG_Content_50kb","GERP.NS","MAF_Adj_LLD_AFR","MAF_Adj_Predicted_Allele_Age","Nucleotide_Diversity_10kb","Recomb_Rate_10kb"]},"yaxis":{"domain":[0,1],"title":"-log10(p-value)","side":"left","range":[0,0.412502783625227]},"yaxis2":{"title":"","showgrid":false,"zeroline":false,"overlaying":"y","side":"right","range":[0,0.412502783625227],"showticklabels":false},"hovermode":"closest"},"source":"A","config":{"modeBarButtonsToAdd":[],"cloud":false,"showLink":false,"displayModeBar":false,"displaylogo":false,"sendData":false},"data":[{"x":["GERP.NS","MAF_Adj_Predicted_Allele_Age","MAF_Adj_LLD_AFR","Recomb_Rate_10kb","Nucleotide_Diversity_10kb","Backgrd_Selection_Stat","CpG_Content_50kb"],"y":[0.195959793135066,0.412502783625227,0.113369145738432,0.00998942626605602,0.0140929706754737,0.22923759400949,0.35360554104753],"hoverinfo":["text","text","text","text","text","text","text"],"text":["Annotation: GERP.NS<br>p-value: 0.637","Annotation: MAF_Adj_Predicted_Allele_Age<br>p-value: 0.387","Annotation: MAF_Adj_LLD_AFR<br>p-value: 0.77","Annotation: Recomb_Rate_10kb<br>p-value: 0.977","Annotation: Nucleotide_Diversity_10kb<br>p-value: 0.968","Annotation: Backgrd_Selection_Stat<br>p-value: 0.59","Annotation: CpG_Content_50kb<br>p-value: 0.443"],"type":"bar","name":"p-value for coefficient","marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","line":{"color":"transparent"}},"xaxis":"x","yaxis":"y","frame":null},{"x":["GERP.NS","MAF_Adj_Predicted_Allele_Age","MAF_Adj_LLD_AFR","Recomb_Rate_10kb","Nucleotide_Diversity_10kb","Backgrd_Selection_Stat","CpG_Content_50kb"],"y":[2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079],"type":"scatter","mode":"lines","hoverinfo":["text","text","text","text","text","text","text"],"text":["","","","","","",""],"name":"Bonferroni significance","yaxis":"y2","line":{"fillcolor":"rgba(255,127,14,1)","color":"rgba(255,127,14,1)"},"xaxis":"x","frame":null},{"x":["GERP.NS","MAF_Adj_Predicted_Allele_Age","MAF_Adj_LLD_AFR","Recomb_Rate_10kb","Nucleotide_Diversity_10kb","Backgrd_Selection_Stat","CpG_Content_50kb"],"y":[1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398],"type":"scatter","mode":"lines","hoverinfo":["text","text","text","text","text","text","text"],"text":["","","","","","",""],"name":"Nominal significance","yaxis":"y2","line":{"fillcolor":"rgba(44,160,44,1)","color":"rgba(44,160,44,1)"},"xaxis":"x","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":[],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
</div>
</div>
</div>
<p>The orange line indicates Bonferroni-corrected significance across 38 annotations for this phenotype (<span class="math inline">\(p &lt;\)</span> 1.3e-3). The green line indicates nominal significance (<span class="math inline">\(p &lt;\)</span> .05). Bonferroni correction excludes the 10 MAF bin annotations and the 26 buffer annotations (extending 500-bp around the baseline functional annotations) since they are primarily intended as statistical controls rather than for direct inference.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
