<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Heritability for 20002_1550</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<script src="site_libs/htmlwidgets-0.9/htmlwidgets.js"></script>
<script src="site_libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/crosstalk-1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.1/js/crosstalk.min.js"></script>
<link href="site_libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 61px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 66px;
  margin-top: -66px;
}

.section h2 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h3 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h4 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h5 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h6 {
  padding-top: 66px;
  margin-top: -66px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">UKBB Heritability</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="https://nealelab.github.io/UKBB_ldsc/index.html">Home</a>
</li>
<li>
  <a href="https://nealelab.github.io/UKBB_ldsc/h2_browser.html">Browser</a>
</li>
<li>
  <a href="https://nealelab.github.io/UKBB_ldsc/details.html">Technical Details</a>
</li>
<li>
  <a href="https://github.com/Nealelab/UKBB_ldsc">Github Repo</a>
</li>
<li>
  <a href="http://www.nealelab.is/">Neale Lab UKBB</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Heritability for 20002_1550</h1>
<h4 class="date"><em>Last updated 2017-09-14</em></h4>

</div>


<p><br></p>
<div class="panel panel-primary">
<div class="panel-body">
<h3 id="phenotype-non-cancer-illness-code-self-reported-lichen-sclerosis"><strong>Phenotype:</strong> Non-cancer illness code, self-reported: lichen sclerosis</h3>
<h4 id="estimated-snp-heritability-0.2185-se0.443-p3.110e-01"><strong>Estimated SNP heritability:</strong> 0.2185 (se=0.443, p=3.110e-01)</h4>
<p>This phenotype can be found on the UK Biobank Showcase for <a href="http://biobank.ctsu.ox.ac.uk/crystal/search.cgi?srch=20002">code 20002</a>. GWAS results are available for <em>N</em> = 337,159 unrelated individuals of European ancestry. This is a binary phenotype with 114 cases and 337,045 controls.</p>
</div>
</div>
<div class="panel panel-info">
<div class="panel-body">
<p class="text-info">
<strong>Note:</strong>
</p>
Heritability for this binary outcome is reported on the liability scale, assuming that the population prevalence matches the observed prevalence of 0.000338 in the UK Biobank analysis set. This may be unreliable if the outcome was only defined for a subset of individuals, or if the the UK Biobank study population (which skews older, whiter, and higher SES than the general UK population, <a href="https://academic.oup.com/aje/article/doi/10.1093/aje/kwx246/3883629/Comparison-of-Sociodemographic-and-Health-Related">among other things</a>) is not a representative sample for this phenotype. For comparison, estimated <strong>observed-scale heritability</strong> is: 0.0010 (se=0.002). The p-value is unaffected.
</div>
</div>
<div class="panel panel-warning">
<div class="panel panel-heading">
<h3 class="panel-title">
Warning
</h3>
</div>
<div class="panel-body">
The effective sample size for this phenotype is quite limited (N_eff=455.85). Estimates are likely to be unstable. Liability-scale transformation for this binary phenotype may be especially misleading.
</div>
</div>
<p><br></p>
<div id="full-results" class="section level3">
<h3>Full Results</h3>
<p>All results are from partitioned heritability analysis of this phenotype using LD score regression (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4495769/">Bulik-Sullivan et al. 2015</a>, <a href="https://github.com/bulik/ldsc">github repo</a>) with 74 annotations as described by <a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3954.html">Gazal et al. 2017</a> (also on <a href="http://www.biorxiv.org/content/early/2017/04/03/082024">biorxiv</a>). See <a href="%60details.html%60">Technical Details</a> for more information.</p>
<p><br></p>
<div id="confounding-and-model-misspecification" class="section level4">
<h4>Confounding and Model Misspecification</h4>
<p>In addition to heritability, LD score regression also estimates a intercept term that indexes of population stratification, other counfounding, and potential misspecification in the partitioned LD score model for the distribution of genetic effects genome-wide.</p>
<ul>
<li><strong>Intercept:</strong> 0.9925 (se=0.00838, p=8.152e-01)</li>
<li><strong>Mean <span class="math inline">\(\chi^2\)</span>:</strong> 0.9905</li>
<li><strong><span class="math inline">\(\lambda_{GC}\)</span>:</strong> 0.9944</li>
<li><strong>Ratio:</strong> NA (se=NA)</li>
</ul>
<p>Intercept values near 1 indicate little or no confounding. Comparing the intercept and the mean <span class="math inline">\(\chi^2\)</span> statistic provides an rough index for how much of the polygenic signal in the GWAS may be due to confounding rather than genetic effects (assuming the LD score model is well specified).</p>
<p><br></p>
</div>
<div id="partitioned-heritability" class="section level4">
<h4>Partitioned Heritability</h4>
<div id="significance-of-categorical-annotations" class="section level5">
<h5>Significance of categorical annotations</h5>
<div id="7c636559fb3d" style="width:800px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="7c636559fb3d">{"x":{"visdat":{"7c6350aa6ef2":["function () ","plotlyVisDat"]},"cur_data":"7c6350aa6ef2","attrs":{"7c6350aa6ef2":{"alpha":1,"sizes":[10,100],"x":{},"y":{},"hoverinfo":"text","text":{},"type":"bar","name":"p-value for coefficient"},"7c6350aa6ef2.1":{"alpha":1,"sizes":[10,100],"x":{},"y":[2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079],"type":"scatter","mode":"lines","hoverinfo":"text","text":"","name":"Bonferroni significance","yaxis":"y2"},"7c6350aa6ef2.2":{"alpha":1,"sizes":[10,100],"x":{},"y":[1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398],"type":"scatter","mode":"lines","hoverinfo":"text","text":"","name":"Nominal significance","yaxis":"y2"}},"layout":{"width":800,"margin":{"b":150,"l":100,"t":25,"r":10},"showlegend":false,"xaxis":{"domain":[0,1],"title":"Annotation","tickangle":-30,"type":"category","categoryorder":"array","categoryarray":["Coding_UCSC","Conserved_LindbladToh","CTCF_Hoffman","DGF_ENCODE","DHS_peaks_Trynka","DHS_Trynka","Enhancer_Andersson","Enhancer_Hoffman","FetalDHS_Trynka","GERP.RSsup4","H3K27ac_Hnisz","H3K27ac_PGC2","H3K4me1_peaks_Trynka","H3K4me1_Trynka","H3K4me3_peaks_Trynka","H3K4me3_Trynka","H3K9ac_peaks_Trynka","H3K9ac_Trynka","Intron_UCSC","Promoter_UCSC","PromoterFlanking_Hoffman","Repressed_Hoffman","Super_Enhancer_Vahedi","SuperEnhancer_Hnisz","TFBS_ENCODE","Transcr_Hoffman","TSS_Hoffman","Typical_Enhancer_Vahedi","UTR_3_UCSC","UTR_5_UCSC","WeakEnhancer_Hoffman"]},"yaxis":{"domain":[0,1],"title":"-log10(p-value)","side":"left","range":[0,1.73378879002159]},"yaxis2":{"title":"","showgrid":false,"zeroline":false,"overlaying":"y","side":"right","range":[0,1.73378879002159],"showticklabels":false},"hovermode":"closest"},"source":"A","config":{"modeBarButtonsToAdd":[],"cloud":false,"showLink":false,"displayModeBar":false,"displaylogo":false,"sendData":false},"data":[{"x":["Coding_UCSC","Conserved_LindbladToh","CTCF_Hoffman","DGF_ENCODE","DHS_peaks_Trynka","DHS_Trynka","Enhancer_Andersson","Enhancer_Hoffman","FetalDHS_Trynka","H3K27ac_Hnisz","H3K27ac_PGC2","H3K4me1_peaks_Trynka","H3K4me1_Trynka","H3K4me3_peaks_Trynka","H3K4me3_Trynka","H3K9ac_peaks_Trynka","H3K9ac_Trynka","Intron_UCSC","PromoterFlanking_Hoffman","Promoter_UCSC","Repressed_Hoffman","SuperEnhancer_Hnisz","TFBS_ENCODE","Transcr_Hoffman","TSS_Hoffman","UTR_3_UCSC","UTR_5_UCSC","WeakEnhancer_Hoffman","Super_Enhancer_Vahedi","Typical_Enhancer_Vahedi","GERP.RSsup4"],"y":[0.791738777584052,1.43567748641907,1.73378879002159,0.0873965879651599,0.434859164161235,0.266177085243934,0.65368315599257,1.07567005390567,0.0187708177163073,0.155368346362262,1.143369006273,0.102557908698108,0.594187255189945,0.00557817790099222,0.254287008415699,1.07595298435228,0.931736009943595,0.225860723918502,0.018815685637565,0.20689890858348,0.887115781477257,0.394209696362661,0.923307178702945,0.154604301981704,0.616142481807963,1.03646147769117,0.0433709192038248,0.331771867608221,0.780317177244211,1.38450857390905,0.787777098736622],"hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"text":["Annotation: Coding_UCSC<br>p-value: 0.162","Annotation: Conserved_LindbladToh<br>p-value: 0.0367","Annotation: CTCF_Hoffman<br>p-value: 0.0185","Annotation: DGF_ENCODE<br>p-value: 0.818","Annotation: DHS_peaks_Trynka<br>p-value: 0.367","Annotation: DHS_Trynka<br>p-value: 0.542","Annotation: Enhancer_Andersson<br>p-value: 0.222","Annotation: Enhancer_Hoffman<br>p-value: 0.084","Annotation: FetalDHS_Trynka<br>p-value: 0.958","Annotation: H3K27ac_Hnisz<br>p-value: 0.699","Annotation: H3K27ac_PGC2<br>p-value: 0.0719","Annotation: H3K4me1_peaks_Trynka<br>p-value: 0.79","Annotation: H3K4me1_Trynka<br>p-value: 0.255","Annotation: H3K4me3_peaks_Trynka<br>p-value: 0.987","Annotation: H3K4me3_Trynka<br>p-value: 0.557","Annotation: H3K9ac_peaks_Trynka<br>p-value: 0.084","Annotation: H3K9ac_Trynka<br>p-value: 0.117","Annotation: Intron_UCSC<br>p-value: 0.594","Annotation: PromoterFlanking_Hoffman<br>p-value: 0.958","Annotation: Promoter_UCSC<br>p-value: 0.621","Annotation: Repressed_Hoffman<br>p-value: 0.13","Annotation: SuperEnhancer_Hnisz<br>p-value: 0.403","Annotation: TFBS_ENCODE<br>p-value: 0.119","Annotation: Transcr_Hoffman<br>p-value: 0.7","Annotation: TSS_Hoffman<br>p-value: 0.242","Annotation: UTR_3_UCSC<br>p-value: 0.0919","Annotation: UTR_5_UCSC<br>p-value: 0.905","Annotation: WeakEnhancer_Hoffman<br>p-value: 0.466","Annotation: Super_Enhancer_Vahedi<br>p-value: 0.166","Annotation: Typical_Enhancer_Vahedi<br>p-value: 0.0413","Annotation: GERP.RSsup4<br>p-value: 0.163"],"type":"bar","name":"p-value for coefficient","marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","line":{"color":"transparent"}},"xaxis":"x","yaxis":"y","frame":null},{"x":["Coding_UCSC","Conserved_LindbladToh","CTCF_Hoffman","DGF_ENCODE","DHS_peaks_Trynka","DHS_Trynka","Enhancer_Andersson","Enhancer_Hoffman","FetalDHS_Trynka","H3K27ac_Hnisz","H3K27ac_PGC2","H3K4me1_peaks_Trynka","H3K4me1_Trynka","H3K4me3_peaks_Trynka","H3K4me3_Trynka","H3K9ac_peaks_Trynka","H3K9ac_Trynka","Intron_UCSC","PromoterFlanking_Hoffman","Promoter_UCSC","Repressed_Hoffman","SuperEnhancer_Hnisz","TFBS_ENCODE","Transcr_Hoffman","TSS_Hoffman","UTR_3_UCSC","UTR_5_UCSC","WeakEnhancer_Hoffman","Super_Enhancer_Vahedi","Typical_Enhancer_Vahedi","GERP.RSsup4"],"y":[2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079,2.88081359228079],"type":"scatter","mode":"lines","hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"text":["","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],"name":"Bonferroni significance","yaxis":"y2","line":{"fillcolor":"rgba(255,127,14,1)","color":"rgba(255,127,14,1)"},"xaxis":"x","frame":null},{"x":["Coding_UCSC","Conserved_LindbladToh","CTCF_Hoffman","DGF_ENCODE","DHS_peaks_Trynka","DHS_Trynka","Enhancer_Andersson","Enhancer_Hoffman","FetalDHS_Trynka","H3K27ac_Hnisz","H3K27ac_PGC2","H3K4me1_peaks_Trynka","H3K4me1_Trynka","H3K4me3_peaks_Trynka","H3K4me3_Trynka","H3K9ac_peaks_Trynka","H3K9ac_Trynka","Intron_UCSC","PromoterFlanking_Hoffman","Promoter_UCSC","Repressed_Hoffman","SuperEnhancer_Hnisz","TFBS_ENCODE","Transcr_Hoffman","TSS_Hoffman","UTR_3_UCSC","UTR_5_UCSC","WeakEnhancer_Hoffman","Super_Enhancer_Vahedi","Typical_Enhancer_Vahedi","GERP.RSsup4"],"y":[1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398,1.30102999566398],"type":"scatter","mode":"lines","hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"text":["","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],"name":"Nominal significance","yaxis":"y2","line":{"fillcolor":"rgba(44,160,44,1)","color":"rgba(44,160,44,1)"},"xaxis":"x","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":[],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p>The orange line indicates Bonferroni-corrected significance across 38 annotations for this phenotype (<span class="math inline">\(p &lt;\)</span> 1.3e-3). The green line indicates nominal significance (<span class="math inline">\(p &lt;\)</span> .05). Bonferroni correction excludes the 10 MAF bin annotations and the 26 buffer annotations (extending 500-bp around the baseline functional annotations) since they are primarily intended as statistical controls rather than for direct inference.</p>
</div>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
