<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Heritability of &gt;2,000 traits &amp; disorders in UK Biobank</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link rel="apple-touch-icon-precomposed" sizes="57x57" href="apple-touch-icon-57x57.png" />
<link rel="apple-touch-icon-precomposed" sizes="114x114" href="apple-touch-icon-114x114.png" />
<link rel="apple-touch-icon-precomposed" sizes="72x72" href="apple-touch-icon-72x72.png" />
<link rel="apple-touch-icon-precomposed" sizes="144x144" href="apple-touch-icon-144x144.png" />
<link rel="apple-touch-icon-precomposed" sizes="60x60" href="apple-touch-icon-60x60.png" />
<link rel="apple-touch-icon-precomposed" sizes="120x120" href="apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon-precomposed" sizes="76x76" href="apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon-precomposed" sizes="152x152" href="apple-touch-icon-152x152.png" />
<link rel="icon" type="image/png" href="favicon-196x196.png" sizes="196x196" />
<link rel="icon" type="image/png" href="favicon-96x96.png" sizes="96x96" />
<link rel="icon" type="image/png" href="favicon-32x32.png" sizes="32x32" />
<link rel="icon" type="image/png" href="favicon-16x16.png" sizes="16x16" />
<link rel="icon" type="image/png" href="favicon-128.png" sizes="128x128" />
<meta name="application-name" content="&nbsp;"/>
<meta name="msapplication-TileColor" content="#FFFFFF" />
<meta name="msapplication-TileImage" content="mstile-144x144.png" />
<meta name="msapplication-square70x70logo" content="mstile-70x70.png" />
<meta name="msapplication-square150x150logo" content="mstile-150x150.png" />
<meta name="msapplication-wide310x150logo" content="mstile-310x150.png" />
<meta name="msapplication-square310x310logo" content="mstile-310x310.png" />


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-106608402-1', 'auto');
  ga('send', 'pageview');
  ga('set', 'anonymizeIp', true);

</script>

<script>
/**
* Function that tracks a click on an outbound link in Analytics.
# modifications from: https://hacks.mozilla.org/2016/01/google-analytics-privacy-and-event-tracking/
*/
var trackOutboundLink = function(url) {
    // check if the GA object exists and that it has initialized
    if(window.ga && ga.loaded) {
        // if yes, rely on GA to follow link
        ga('send', 'event', 'outbound', 'click', url, {
            'transport': 'beacon',
            'hitCallback': function(){document.location = url;}
        });
    } else {
        // if not, follow link ourselves
        document.location = url;
    }
}
</script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 61px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 66px;
  margin-top: -66px;
}

.section h2 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h3 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h4 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h5 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h6 {
  padding-top: 66px;
  margin-top: -66px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">UKBB Heritability</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="h2_browser.html">Browser</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Plots
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="viz_h2.html">Heritability and Intercept</a>
    </li>
    <li>
      <a href="viz_annot.html">Partitioned Annotations</a>
    </li>
    <li>
      <a href="viz_qq.html">QQ Plots</a>
    </li>
    <li>
      <a href="viz_sampsize.html">Sample Size Scaling</a>
    </li>
    <li>
      <a href="viz_univar.html">Univariate LDSR</a>
    </li>
  </ul>
</li>
<li>
  <a href="details.html">Technical Details</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/Nealelab/UKBB_ldsc">Github Repo</a>
</li>
<li>
  <a href="http://www.nealelab.is/blog/2017/7/19/rapid-gwas-of-thousands-of-phenotypes-for-480000-samples-in-the-uk-biobank">Neale Lab UKBB</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Heritability of &gt;2,000 traits &amp; disorders in UK Biobank</h1>

</div>


<p><br></p>
<div class="well">
<p>As you may know, the UK Biobank recently released data on ~500,000 individuals to the scientific community to advance research on genetics. The <a href="http://nealelab.is">Neale Lab</a> has been working to make some basic analyses of that data freely available; you can read more about the overall effort <a href="http://www.nealelab.is/blog/2017/7/19/rapid-gwas-of-thousands-of-phenotypes-for-480000-samples-in-the-uk-biobank">here</a>.</p>
<p><br></p>
<p>Building on the initial genome-wide association analyses, we’ve now estimated the heritability of each of those traits and disorders, over 2,000 as of this writing. As with the primary analyses, we’re making the heritability results available for <a href="h2_browser.html">browsing</a> and bulk <a href="https://www.dropbox.com/sh/g28ee03vy4hvqw5/AADAkDbSFHsYE8bme1Jjeekca/ldsc_results?dl=0" onclick="trackOutboundLink('https://www.dropbox.com/sh/g28ee03vy4hvqw5/AADAkDbSFHsYE8bme1Jjeekca/ldsc_results?dl=0'); return false;">download</a>.</p>
<p><br></p>
<p><a href="h2_browser.html" class="btn btn-primary btn-lg">Browse <span class="math inline">\(h^2\)</span></a> <a href="plots_home.html" class="btn btn-primary btn-lg">Explore Plots</a> <a href="https://www.dropbox.com/sh/g28ee03vy4hvqw5/AADAkDbSFHsYE8bme1Jjeekca/ldsc_results?dl=0"  class="btn btn-primary btn-lg" onclick="trackOutboundLink('https://www.dropbox.com/sh/g28ee03vy4hvqw5/AADAkDbSFHsYE8bme1Jjeekca/ldsc_results?dl=0'); return false;">Download Results</a></p>
<p><br></p>
<p>You can learn more about the heritability analysis in this series of blog posts:</p>
<ul>
<li><a href="http://www.nealelab.is/blog/2017/9/13/heritability-101-what-is-heritability">Heritability 101: What is “heritability”?</a></li>
<li><a href="http://www.nealelab.is/blog/2017/9/13/heritability-201-types-of-heritability-and-how-we-estimate-it">Heritability 201: Types of heritability and how we estimate it</a></li>
<li>(coming soon) Insights from estimates of SNP-heritability for &gt;2,000 traits and disorders in UK Biobank</li>
</ul>
<p><br></p>
<p>More technical descriptions are also available:</p>
<ul>
<li><a href="http://www.nealelab.is/blog/2017/9/14/heritability-501-ldsr-based-h2-in-ukbb-for-the-technically-minded">Heritability 501: LDSR-based h2 in UKBB for the technically-minded</a></li>
<li><a href="https://github.com/Nealelab/UKBB_ldsc">UKBB_ldsc github repository</a></li>
</ul>
<p><br></p>
<p>We view these results as preliminary and subject to change as we continue to refine the analyses as well as the genome-wide association results they are based on. But we hope you’ll agree that they provide an interesting and useful first look at the genetics of the many diverse traits and disorders studied in the UK Biobank.</p>
<p><br></p>
<p>If you want more information, please <strong>contact us at: <a href="mailto:nealelab.ukb@gmail.com">nealelab.ukb@gmail.com</a>.</strong></p>
</div>
<p><br> <br></p>
<div class="well">
<h4 id="faq">FAQ</h4>
<p><strong>Where can I find the heritability results?</strong></p>
<p>We’ve created a browser to explore the results <a href="h2_browser.html">here</a>. You can also download the full results file <a href="https://www.dropbox.com/sh/g28ee03vy4hvqw5/AADAkDbSFHsYE8bme1Jjeekca/ldsc_results?dl=0" onclick="trackOutboundLink('https://www.dropbox.com/sh/g28ee03vy4hvqw5/AADAkDbSFHsYE8bme1Jjeekca/ldsc_results?dl=0'); return false;">here</a>.</p>
<p><br></p>
<p><strong>What method have you used to estimated heritability?</strong></p>
<p>To enable this quick initial analysis on such large-scale data we’re relying on LD score regression (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4495769/">article</a>, <a href="https://github.com/bulik/ldsc">github repo</a>). Full code and technical details are available in the <a href="https://github.com/Nealelab/UKBB_ldsc">UKBB_ldsc github repository</a> and described in <a href="http://www.nealelab.is/blog/2017/9/14/heritability-501-ldsr-based-h2-in-ukbb-for-the-technically-minded">Heritability 501</a>.</p>
<p><br></p>
<p><strong>Are the primary GWAS results used for this analysis available? </strong></p>
<p>Yes! Results will be available for browsing through the <a href="https://biobankengine.stanford.edu/">Global Biobank Engine</a>, and the full GWAS results are available for download from <a href="https://www.dropbox.com/sh/g28ee03vy4hvqw5/AAAGunSi-VK-0cUqvcwyt67ra/associations?dl=0">here</a>. See <a href="http://www.nealelab.is/blog/2017/7/19/rapid-gwas-of-thousands-of-phenotypes-for-480000-samples-in-the-uk-biobank">the introductory post</a> for much more information.</p>
<p><br></p>
<p><strong>How should I interpret these heritability results?</strong></p>
<p><em>Very carefully.</em> For starters, we’ve written a couple of primers that cover the concept of heritability and common misconceptions (<a href="http://www.nealelab.is/blog/2017/9/13/heritability-101-what-is-heritability">Heritability 101</a>) and the specific type of heritability that’s being estimated in this analysis (<a href="http://www.nealelab.is/blog/2017/9/13/heritability-201-types-of-heritability-and-how-we-estimate-it">Heritability 201</a>). A longer discussion of the many limitations of this analysis is included in our more technical description of the project (<a href="http://www.nealelab.is/blog/2017/9/14/heritability-501-ldsr-based-h2-in-ukbb-for-the-technically-minded">Heritability 501</a>).</p>
<p>We’ll revisit this question in future posts as we explore the results in more detail, but here’s the executive summary: This is a very rough initial analysis and should be treated accordingly. All estimates are for (a) the heritability from common genetic variants (b) of automatically cleaned and transformed phenotypes (c) in a non-random population sample of the UK (d) estimated with a fairly simple statistical model. We expect the results will mostly be useful for generating hypotheses for follow-up (e.g. identifying trends in the estimates for categories of phenotypes, choosing phenotypes with robust statistical evidence for heritability), rather than being a definitive statement about the “true” heritability of a individual phenotype.</p>
</div>
<p><br> <br></p>
<div class="well">
<h4 id="credits">Credits</h4>
<p><strong>Heritability analysis team:</strong> Liam Abbott, Sam Bryant, Claire Churchhouse, Benjamin Neale, Duncan Palmer, Raymond Walters</p>
<p><strong>Neale Lab UKBB team:</strong> Liam Abbott, Verneri Anttila, Krishna Aragam, Jon Bloom, Sam Bryant, Claire Churchhouse, Joanne Cole, Mark J. Daly, Andrea Ganna, Jackie Goldstein, Mary Haas, Joel Hirschhorn, Daniel Howrigan, Sekar Kathiresan, Dan King, Benjamin Neale, Duncan Palmer, Tim Poterba, Manuel Rivas, Cotton Seed, Sailaja Vedantam, Raymond Walters</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
